server/models/UserProfile.js

```diff
@@ -0,0 +1,36 @@
+const mongoose = require('mongoose')
+const reqString = {
+  type: 'string',
+  required: true
+}
+
+const noReqString = {
+  type: 'string',
+  required: false
+}
+
+const userProfile = new mongoose.Schema(
```

Comment:
Don't we need an availability field too? :)

Reply:
i thought the "availability field" is the same thing as online Status.

Reply:
The `availability` field contains information about a dog sitters availability (days of the week and the period of time  they are available for service)

On a side note, you can remove the `online` field once this is done.

Reply:
ok